packages/react-native-editor/src/api-fetch-setup.js

```diff
@@ -81,8 +81,18 @@ export const isPathSupported = ( path, method ) => {
 	);
 };
 
-export const shouldEnableCaching = ( path ) =>
-	! DISABLED_CACHING_ENDPOINTS.some( ( pattern ) => pattern.test( path ) );
+export const shouldEnableCaching = ( path ) => {
+	const disabledCachingEndpoints = applyFilters(
+		'native.disabled_caching_endpoints',
+		DISABLED_CACHING_ENDPOINTS
+	);
+
+	const isDisabled = disabledCachingEndpoints.some( ( pattern ) =>
```

Comment:
I was getting a linting complaint about trying to return `! disabledCachingEndpoints.som(...)` . I suppose I could have wrapped the negated return in parentheses but I opted for assigning the result to a variable which makes the intent clearer to me.